pear.js

```diff
@@ -28,11 +28,14 @@ const BIN = path.join(PEAR_DIR, 'bin')
 const CURRENT_BIN = path.join(LINK, 'by-arch', HOST, 'bin/pear-runtime' + (isWindows ? '.exe' : ''))
 
 if (isInstalled()) {
-  const warning = `[ WARNING ] To complete Pear installation, prepend the following to the system ${isWindows ? 'Path environment variable' : '$PATH'}:
-${BIN}
-Until then, this executable spawns the ${'`pear`'} binary.
-Fix automatically with: pear run pear://runtime`
-  console.error(warning)
+  const systemPath = process.env.PATH || ''
+  if (!systemPath.includes(BIN)) {
```

Comment:
The system path not only needs the bin to be included, it also needs that is included before the npm path. Also if the bin is configured correctly, calling `pear` would not execute this code at all, because the bin would be included in the path already, so in my opinion this check is not necessary.